CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 0.5.0
+
+* Default to JRuby 9.1.10.0 [#30]
```

Comment:
Agreed, that doesn't make any real difference to anyone not developing against it.